c2corg_ui/templates/i18n.html

```diff
@@ -439,3 +439,75 @@
 ## misc
 <span translate>see more results</span>
 <span translate>Associated waypoint_children</span>
+
+## event_types
+<span translate>avalanche</span>
+<span translate>stone_fall</span>
+<span translate>falling_ice</span>
+<span translate>person_fall</span>
+<span translate>crevasse_fall</span>
+<span translate>roped_fall</span>
+<span translate>physical_failure</span>
+<span translate>lightning</span>
+<span translate>other</span>
+
+## author_statuses
+<span translate>primary_impacted</span>
+<span translate>secondary_impacted</span>
+<span translate>internal_witness</span>
+<span translate>external_witness</span>
+
+## activity_rates
+<span translate>activity_rate_150</span>
+<span translate>activity_rate_50</span>
+<span translate>activity_rate_30</span>
+<span translate>activity_rate_20</span>
+<span translate>activity_rate_10</span>
+<span translate>activity_rate_5</span>
+<span translate>activity_rate_1</span>
+
+## nb_outings
+<span translate>nb_outings_4</span>
+<span translate>nb_outings_9</span>
+<span translate>nb_outings_14</span>
+<span translate>nb_outings_15</span>
+
+## genders
+<span translate>male</span>
+<span translate>female</span>
+
+## previous_injuries
+<span translate>no</span>
+<span translate>previous_injuries_2</span>
+<span translate>previous_injuries_3</span>
+
+## severities
+<span translate>severity_no</span>
+<span translate>1d_to_3d</span>
+<span translate>4d_to_1m</span>
+<span translate>1m_to_3m</span>
+<span translate>more_than_3m</span>
+
+## autonomies
+<span translate>non_autonomous</span>
+<span translate>autonomous</span>
+<span translate>initiator</span>
+<span translate>expert</span>
+
+## avalanche_levels
+<span translate>level_1</span>
```

Comment:
I'm afraid those strings are a bit too generic. Using ``avalanche_risk_1`` for instance would be less confusing (including for translators).